src/pages/index.tsx

```diff
@@ -6,27 +6,42 @@ import Layout from '@theme/Layout';
 import HomepageFeatures from '@site/src/components/HomepageFeatures';
 
 import styles from './index.module.css';
+import Pic from '../../static/img/mainPic.png';
 
 function HomepageHeader() {
   const {siteConfig} = useDocusaurusContext();
   return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
-      <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
+    <header className={clsx('hero hero--primary', styles.heroBanner)} style={{backgroundImage:'linear-gradient(to top, rgba(230, 239, 255, 1) , rgba(255, 255, 255, 1))',textAlign:'left',fontWeight:'Semibold',fontFamily:'PingFang SC-Semibold',display:'flex'}}>
+      <div className="container" style={{margin:'0 0 0 10%',width:'40%'}}>
+        <h1 className="hero__title" style={{color:'black'}}>{siteConfig.title}</h1>
+        <h2 className='hero__secondary__title' style={{color:'rgba(40, 117, 243, 1)'}}>(incubating)</h2>
+        <div className="hero__subtitle" style={{color:'rgba(59, 61, 63, 1)',fontFamily:'PingFang SC-Light',fontWeight:'Light',width:'30rem',marginTop:'2rem'}}>{siteConfig.tagline}</div>
```

Comment:
AFAIK PingFang FC is a proprietary font so maybe we cannot use it in our site.

Reply:
I will talk to the UI designer about font.

Reply:
Thanks. Another concern from me is that, in one hand, maybe we do not need a CJK font since there is nearly no Chinese words (only English) in our site; and in other hand, maybe we should include the font (stored) in our site (e.g. in WOFF format) if we really want a custom English font.